perfkitbenchmarker/providers/openstack/os_virtual_machine.py

```diff
@@ -197,6 +267,43 @@ def DeleteKeyfile(self):
         except NotFound:
             logging.info("Deleting key doesn't exists")
 
+    def _GetBlockDevices(self):
+        stdout, _ = self.RemoteCommand(
+            'sudo lsblk -o NAME,MODEL,SIZE,TYPE,MOUNTPOINT,LABEL -n -b -P')
+        lines = stdout.splitlines()
+        groups = [LSBLK_PATTERN.match(line) for line in lines]
+        tuples = [g.groups() for g in groups if g]
+        colnames = ('name', 'model', 'size_bytes',
+                    'type', 'mountpoint', 'label',)
+        blk_devices = [dict(zip(colnames, t)) for t in tuples]
+        for d in blk_devices:
+            d['model'] = d['model'].rstrip()
+            d['label'] = d['label'].rstrip()
+            d['size_bytes'] = int(d['size_bytes'])
+        return blk_devices
+
+    def _GetBootDevice(self):
```

Comment:
This one as well. Once this PR gets merged, on a separate branch this can refactored to the `BaseLinuxMixin` class. Since it's duplicated on `rackspace_virtual_machine.py` as well.